releases/release-1.22/release-notes/maps/pr-102516-map.yaml

```diff
@@ -0,0 +1,4 @@
+pr: 102516
+releasenote:
+  text: '''serviceOwnsFrontendIP'' shouldn''t report error when the public IP doesn''t
```

Comment:
It should start with : 
The `serviceOwnsFrontendIP` and there are one to many '' at should and does not